ci/infra/libvirt/terraform.tfvars.json.ci.example

```diff
@@ -1,9 +1,9 @@
 {
-    "libvirt_uri": "qemu:///system",
+    "libvirt_uri": "qemu+ssh://jenkins@node52.cloud.suse.de/system",
```

Comment:
I am wondering if this should be part of configuration options in `vars.yml` or part of an ENV file

Reply:
I would say vars, because that gives us automatic ENV vars so its easy to override and easy to track (both in file and easily overridable)

The problem I see here is, we can only manage a kvm host at once? So to deploy on kvm we need a huge machine to be able to deploy caasp in just one? How does that work for customers wanting to deploy in several servers? They have to keep changing the kvm host? How does networking work with that?

Reply:
`terraform-provider-libvirt` is just not supported at all, thus this is out of the scope of customers, this is only for internal use. Terraform plus libvirt tandem is just not delivered to customers, AFAIK we just deliver documentation about how to use CaaSP on top of libvirt (something close to bare metal procedure), not any sort of automation.

Reply:
Perfect!